### DISCUSSION TITLE
Combo box: text field with a drop-down

### DISCUSSION THREAD

**Discussion by JoseGonzalz** [Telefonica/mistica-design#1931] [Proposals]
### Goal

Make it easy to select in a drop-down with a large number of options.

### Context

We have a form to fill in the address and the users have to select their country. This selection has a large number of options (almost 200 countries) and they have to make drop-down to select one.

### Impact

Create a new experience and option to create a combo box for the cases it is necessary to show a large number of options.

### Discussion Questions

- Could we have this new combo box in the assets?
- Are there other options to do this?
- Do you recommend creating another experience with this component?

### Do you have a visual proposal?

https://www.figma.com/design/MOlXwoA7H3DuIjbj8JRLbK/%5BDIPA%5D-Add-input-text-in-dropdown-selector-Q3-2024-PRODUCTDSN-2717?m=auto&t=JHe9BRl4gzNhTG7W-6

**Comment by aweell** [comment]
There is already a ticket covering this topic: https://github.com/Telefonica/mistica-design/issues/1662

Prioritize this topic accordingly in your needs in order to start working on it.
